docs/docs-beta/docs/dagster-plus/features/asset-catalog/catalog-views.md

```diff
@@ -46,7 +41,7 @@ To create a new catalog view, you have two options:
 
 ![Screenshot of catalog view customization UI](/img/placeholder.svg)
 
-Give your view a name and optionally a description and icon. Next, you can select one or more filters to apply to your view by clicking the "Add filters" button. Filters can select a subset of assets based on their metadata, tags, kinds, owners, asset groups, or other properties.
+Give your view a name and optionally a description and icon. Next, you can select one or more filters to apply to your view by clicking the "Add filters" button. Filters can select a subset of assets based on their [metadata](/guides/build/assets/organizing-assets-with-tags-and-metadata), tags, kinds, owners, asset groups, or other properties.
```

Comment:
Question. Is our standard that buttons are in quotations and navigation is bold. Just noticed `clicking the "Add filters" button` vs `click **Catalog**` below

Reply:
Good catch, I've been making buttons bold, since that seems to the style in the majority of existing docs and that's what's mentioned in the [existing style guide](https://github.com/dagster-io/dagster/blob/master/docs/DOC_CHECKLIST.md#formatting). I'll update the style here, and also update the style guide to remove quotations from the examples, since that seems a little confusing.